protocol/messenger_installations_test.go

```diff
@@ -250,28 +250,6 @@ func (s *MessengerInstallationSuite) TestSyncInstallation() {
 
 	s.Require().NotNil(statusChat)
 
-	var actualContact *Contact
-	for _, c := range allContacts {
-		if c.ID == contact.ID {
-			actualContact = c
-		}
-	}
-
+	actualContact := allContacts[0]
```

Comment:
Why are we just picking the first one here? I assume it's the only one with custom fields, but could we make this more explicit?

Reply:
@jakubgs how would you propose making this more explicit? Adding a comment? Changing the variable name?

Reply:
@cammellos is there a check that the `allContacts` slice has a len > 0?

Reply:
Yes, we would throw an error if the slice has length `0` (in `RetryWithBackOff` we return an error until `len(x) == 1`)